go.mod

```diff
@@ -2,34 +2,37 @@ module k8s.io/kops
 
 go 1.14
 
-// Version kubernetes-1.20.0-beta.2 => tag v0.20.0
+// Version kubernetes-1.20.4 => tag v0.20.4
```

Comment:
Please remove this comment.

Reply:
Don't have to update it at least. The comment is nice enough as I always forget to remove the 1.

Reply:
Easy to see that it break things later :)

Reply:
@hakman 
Are you happy for me to leave it in?
I agree with @olemarkus that it's good to see how most of the `k8s.io` Go modules versioning relates to the release versioning of Kubernetes. It certainly helped me when I started working on the recent PRs.

Reply:
I think so, just please move the other on the top.